pkg/service/repair/repair_test.go

```diff
@@ -132,148 +147,142 @@ func TestShouldRepairRing(t *testing.T) {
 	}{
 		{
 			Ring: scyllaclient.Ring{
-				HostDC:      hostDC,
-				Replication: scyllaclient.LocalStrategy,
-				RF:          1,
+				ReplicaTokens: []scyllaclient.ReplicaTokenRanges{rs("h1")},
+				HostDC:        hostDC,
 			},
-			DCs:      []string{"dc1", "dc2", "dc3", "dc4"},
+			DCs:      allDCs,
 			Expected: false,
 		},
 		{
 			Ring: scyllaclient.Ring{
-				HostDC:      hostDC,
-				Replication: scyllaclient.SimpleStrategy,
-				RF:          1,
+				ReplicaTokens: []scyllaclient.ReplicaTokenRanges{
+					rs("h1"), rs("h2"), rs("h3"),
+					rs("h4"), rs("h5"), rs("h6"),
+					rs("h7"), rs("h8"), rs("h9"),
+				},
+				HostDC: hostDC,
 			},
-			DCs:      []string{"dc1", "dc2", "dc3", "dc4"},
+			DCs:      allDCs,
 			Expected: false,
 		},
 		{
 			Ring: scyllaclient.Ring{
-				HostDC:      hostDC,
-				Replication: scyllaclient.SimpleStrategy,
-				RF:          2,
+				ReplicaTokens: []scyllaclient.ReplicaTokenRanges{
+					rs("h1", "h2"), rs("h2", "h3"), rs("h3", "h4"),
+					rs("h4", "h5"), rs("h5", "h6"), rs("h6", "h7"),
+					rs("h7", "h8"), rs("h8", "h9"), rs("h9", "h1"),
```

Comment:
Is this even possible scenario ? 
Replica sets here are of the same table.

rs("h2", "h3") means that data is replicated in dc2 (that may have RF=1) and in dc3 on one node (what means that RF for this DC is 1 as well), the next rs is ("h3", "h4"), what means that data is on two nodes from dc3 (RF = 2).

Reply:
I basically just converted previous tests based on explicit replication strategy to token rings.
This test here tests for SimpleReplicationStrategy, so that's why replica count per DC is not preserved.